{"name": "test_case_01", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF788213E52+31618]\n\t(No symbol) [0x00007FF78818B0B9]\n\t(No symbol) [0x00007FF78804888A]\n\t(No symbol) [0x00007FF788098524]\n\t(No symbol) [0x00007FF78809862C]\n\t(No symbol) [0x00007FF7880DF787]\n\t(No symbol) [0x00007FF7880BD14F]\n\t(No symbol) [0x00007FF7880DCA80]\n\t(No symbol) [0x00007FF7880BCEB3]\n\t(No symbol) [0x00007FF78808A46B]\n\t(No symbol) [0x00007FF78808B001]\n\tGetHandleVerifier [0x00007FF78851A02D+3202397]\n\tGetHandleVerifier [0x00007FF788566A4D+3516285]\n\tGetHandleVerifier [0x00007FF78855C4C0+3473904]\n\tGetHandleVerifier [0x00007FF7882C5D56+760454]\n\t(No symbol) [0x00007FF788196B5F]\n\t(No symbol) [0x00007FF788191CF4]\n\t(No symbol) [0x00007FF788191E82]\n\t(No symbol) [0x00007FF78818122F]\n\tBaseThreadInitThunk [0x00007FFDDA891F87+23]\n\tRtlUserThreadStart [0x00007FFDDC68E120+32]", "trace": "self = <test_case_01.Test_case_01 object at 0x0000015B7CFDA990>\n\n    @pytest.mark.usefixtures(\"setup_teardown\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_case_01(self):\n    \n        extend_testing = (By.XPATH,\"\")\n    \n        # reading the data here, from test_case_01.json\n        with open(TEST_DATA_DIR + '/test_case_01.json', 'r') as json_file:\n            test_data = json.load(json_file)\n            customer_unit = test_data['customer_unit']\n    \n        wait = WebDriverWait(self.driver, 30)\n    \n        product_browse  = ProductBrowsing(self.driver)\n        product_cart_validate = CartCheckout(self.driver)\n    \n        product_browse.product_browsing()\n    \n        product_name = product_browse.adding_products_to_cart()\n    \n>       product_cart_validate.checkout_process(product_name)\n\nsrc\\tests\\smoke\\test_case_01.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nsrc\\pages\\product_cart.py:28: in checkout_process\n    ???\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0a6296088de9fc18ca3f5e3535a1aeee\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x0000015B7DE42CA0>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       \tGetHandleVerifier [0x00007FF788213E52+31618]\nE       \t(No symbol) [0x00007FF78818B0B9]\nE       \t(No symbol) [0x00007FF78804888A]\nE       \t(No symbol) [0x00007FF788098524]\nE       \t(No symbol) [0x00007FF78809862C]\nE       \t(No symbol) [0x00007FF7880DF787]\nE       \t(No symbol) [0x00007FF7880BD14F]\nE       \t(No symbol) [0x00007FF7880DCA80]\nE       \t(No symbol) [0x00007FF7880BCEB3]\nE       \t(No symbol) [0x00007FF78808A46B]\nE       \t(No symbol) [0x00007FF78808B001]\nE       \tGetHandleVerifier [0x00007FF78851A02D+3202397]\nE       \tGetHandleVerifier [0x00007FF788566A4D+3516285]\nE       \tGetHandleVerifier [0x00007FF78855C4C0+3473904]\nE       \tGetHandleVerifier [0x00007FF7882C5D56+760454]\nE       \t(No symbol) [0x00007FF788196B5F]\nE       \t(No symbol) [0x00007FF788191CF4]\nE       \t(No symbol) [0x00007FF788191E82]\nE       \t(No symbol) [0x00007FF78818122F]\nE       \tBaseThreadInitThunk [0x00007FFDDA891F87+23]\nE       \tRtlUserThreadStart [0x00007FFDDC68E120+32]\n\n..\\..\\..\\..\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException"}, "attachments": [{"name": "log", "source": "fa435c1b-53a4-4125-9a25-f937bda1bafc-attachment.txt", "type": "text/plain"}], "start": 1720020105040, "stop": 1720020218247, "uuid": "8fbfbe88-906d-4e4a-b6a2-6f12ef5d819a", "historyId": "da8a131242ee5a40dd67ae6ebe4a3496", "testCaseId": "da8a131242ee5a40dd67ae6ebe4a3496", "fullName": "src.tests.smoke.test_case_01.Test_case_01#test_case_01", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_teardown')"}, {"name": "parentSuite", "value": "src.tests.smoke"}, {"name": "suite", "value": "test_case_01"}, {"name": "subSuite", "value": "Test_case_01"}, {"name": "host", "value": "LAPTOP-1QRA8EB8"}, {"name": "thread", "value": "1376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.smoke.test_case_01"}]}